components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/ActionExecutionResponseProcessor.java

```diff
@@ -47,6 +49,23 @@ ActionExecutionStatus<Success> processSuccessResponse(Map<String, Object> eventC
                                                           ActionInvocationSuccessResponse successResponse) throws
             ActionExecutionResponseProcessorException;
 
+    /**
+     * This method processes the incomplete response received from the action execution.
```

Comment:
Better mention the default implementation throws an unsupported exception so that actions that requires incomplete state can implement this method